internal/matchers/TemporalMatcher.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package matchers
+
+type TemporalMatcher byte
```

Comment:
nit: I'm in favor of using a lowercase filename, for consistency (?)--e.g., Expectation is in a file named expectation.

Reply:
Totally. Not sure how that came to be, although I blame me.